tv_apps/tv-epg/js/channel_manager.js

```diff
@@ -0,0 +1,244 @@
+/* global evt */
+
+'use strict';
```

Comment:
It looks like this file is totally the same as channel_manager in TV. Should we put them at `shared/smart-screen/tv` folder?